ui/src/app/pages/profile/profile-component.spec.tsx

```diff
@@ -160,13 +162,4 @@ describe('ProfilePageComponent', () => {
       wrapper.find('[data-test-id="signed-ducc-panel"]').exists()
     ).toBeFalsy();
   });
-
-  it('should not display a link to the signed DUCC if the user has signed an older DUCC', async () => {
-    updateProfile({ duccSignedVersion: 3 });
```

Comment:
We could either delete this test or change it to an even older version of DUCC (v2?).

Reply:
yeah I'd like to keep it v2